docs/resources/deployment-examples/example-fargate/terraform/vpc.tf

```diff
@@ -0,0 +1,34 @@
+variable "region" {
+  default     = "us-east-2"
```

Comment:
This is nice! I noticed that you also hardcoded `us-east-2` in a few places. Should we use this variable across all files to allow us to switch region easily?